src/cli/commands/install.js

```diff
@@ -202,6 +202,7 @@ export class Install {
     this.integrityChecker = new InstallationIntegrityChecker(config);
     this.linker = new PackageLinker(config, this.resolver);
     this.scripts = new PackageInstallScripts(config, this.resolver, this.flags.force);
+    this.depMap = new Map(); // Keep track of dependencies for access by key 
```

Comment:
Probably not a good name. Didn't see the other local `depMap` until re-reading during lint fixes.